src/docs/markdown/getting-started.md

```diff
@@ -34,6 +34,10 @@ Let's start by running it:
 
 Oops; without a subcommand, the `caddy` command only displays help text. You can use this any time you forget what to do.
 
+<aside class="tip" markdown="1">
+	Caddy is controlled through the `caddy` command. It offers many useful tools for different tasks while running Caddy. Have a closer look at [ `caddy`'s subcommands ](https://caddyserver.com/docs/command-line) to learn what you can do. 
+</aside>
```

Comment:
Unfortunately, that doesn't work 😢 

![image](https://user-images.githubusercontent.com/2111701/172065459-d3c63d8d-a2a6-441a-8f96-e2e93bfd8784.png)

Might be a good idea to request support for this from https://github.com/yuin/goldmark which is what Caddy uses

Reply:
There's some mention of `markdown=1` in the CommonMark spec FYI https://spec.commonmark.org/0.30/#example-189 but I'll leave it up to you to request support for this

Reply:
Oh wow, this works, actually:

```suggestion
<aside class="tip" markdown="1">

Caddy is controlled through the `caddy` command. It offers many useful tools for different tasks while running Caddy. Have a closer look at [ `caddy`'s subcommands ](https://caddyserver.com/docs/command-line) to learn what you can do. 

</aside>
```

Noticed this from reading that CommonMark section, it mentions using the newlines as an alternative.

![image](https://user-images.githubusercontent.com/2111701/172065719-f58af941-3571-48d3-a381-c0f5676874c0.png)

Reply:
I think I'm surprised that works, although maybe I do remember seeing that it works except something was off, maybe it was the line breaks or something.

Reply:
Paragraphs seem to work fine

![image](https://user-images.githubusercontent.com/2111701/172105609-ccb3f299-41ab-4f5d-ac09-7594fff530c6.png)


```md
<aside class="tip" markdown="1">

Caddy is controlled through the `caddy` command. It offers many useful tools for different tasks while running Caddy. Have a closer look at [ `caddy`'s subcommands ](https://caddyserver.com/docs/command-line) to learn what you can do. 

A new line for testing

Another new line for testing

</aside>
```